lib/licensee/project_files/project_file.rb

```diff
@@ -40,6 +40,7 @@ def initialize(content, metadata = {})
         unless @content.valid_encoding?
           @content.encode!(ENCODING, **ENCODING_OPTIONS)
         end
+        @content.encode!(ENCODING, universal_newline: true)
```

Comment:
Would it make sense to add `universal_newline: true` to the encoding options hash above and always run line 41 instead? Alternatively, I could see this living in content normalizer, replacing `\r\n?` with `\n`

Reply:
I was not able to get adding `universal_newline: true` to the encoding options and always running line 41. Tests involving non-UTF-8-encoded license obtain `ArgumentError: invalid byte sequence in UTF-8`. I'm not sure why, but it seems the operations conflict with each other somehow.

I didn't include int he content normalizer because I'd read (but did not test) that string.encode is much faster than a regex, and it seemed fair to consider a pre-normalization step, having to do with the file type rather than the content.

However if you'd prefer it live in the content normalizer for consistency purposes, happy to move it there.